README.md

```diff
@@ -45,27 +44,53 @@ You can also combine the flag options:
 
 ## Installation
 
-The current stable version of python-magic is available on PyPI and
-can be installed by running `pip install python-magic`.
+This module is a simple [CDLL](https://docs.python.org/3/library/ctypes.html) wrapper around the libmagic C library.
+The current stable version of python-magic is available on [PyPI](http://pypi.python.org/pypi/python-magic/)
+and can be installed by running `pip install python-magic`.
 
-Other sources:
+Compiled libmagic and the magic database come bundled in the wheels on PyPI.
+You can use your own `magic.mgc` database by setting the `MAGIC`
+environment variable, or by using `magic.Magic(magic_file='path/to/magic.mgc')`.
+If you want to compile your own libmagic, circumvent the wheels
+by installing from source: `pip install python-magic --no-binary python-magic`.
 
-- PyPI: http://pypi.python.org/pypi/python-magic/
-- GitHub: https://github.com/ahupp/python-magic
+For systems not supported by the wheels, pip installs from source,
+requiring libmagic to be available before installing python-magic:
 
-This module is a simple wrapper around the libmagic C library, and
-that must be installed as well:
+### Linux
 
-### Debian/Ubuntu
+The Linux wheels should run on most systems out of the box.
 
+Depending on your system and CPU architecture, there might be no compatible wheel uploaded.
+However, precompiled libmagic might still be available for your system:
+
+```sh
+# Debian/Ubuntu
+apt-get update && apt-get install -y libmagic1
+# Alpine
+apk add --update libmagic
+# RHEL
```

Comment:
It would be nice to mention fedora as well since it's more [popular](https://distrowatch.com/dwres.php?resource=popularity). Also I'd suggest to recommend dnf instead as a more modern alternative.

Reply:
> is dnf installed by default? if yes, can you provide the relevant command?

DNF stands for Dandified YUM and is basically an improved version of the YUM package manager.

[Red Hat Doc](https://docs.redhat.com/it/documentation/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index#proc_providing-feedback-on-red-hat-documentation_managing-software-with-the-dnf-tool)
>In Red Hat Enterprise Linux (RHEL) 9, use the DNF utility to manage software. For compatibility reasons with previous major RHEL versions, you can still use the yum command. However, in RHEL 9, yum is an alias for dnf which provides a certain level of compatibility with yum.

[yum-dnf manual from opensource.com](https://opensource.com/article/18/8/guide-yum-dnf)
> [about DNF] It was first introduced in Fedora 18 in a testable state (i.e., tech preview), but it's been Fedora's default package manager since Fedora 22.

The command is the same:
`dnf install SAME_PACKAGE_AS_FOR_YUM`